app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -31,3 +32,22 @@ const deserialize = (data) => {
     stories: projectBoard.stories.map(Story.deserialize)
   }
 }
+
+export const hasSearch = projectBoard => Boolean(projectBoard.search.keyWord);
+
+export const validSearch = keyWord => Boolean(keyWord.trim());
+
+export const isOperandSearch = word => {
+  const operandsWords = Object.keys(operands);
+
+  return operandsWords.some(operand => word.includes(operand)) && word.includes(':');
```

Comment:
You could create a new method for `operandsWords.some(operand => word.includes(operand))`.

```js
const constainsInArray = (word, array) =>  array.some(operand => word.includes(operand))

export const isOperandSearch = word => 
  constainsInArray(word, Object.keys(operands)) && word.includes(':')
```